packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -88,33 +219,38 @@ chmod +x %{buildroot}%{gem_instdir}/extra/dynflow-debug.sh
 ln -s %{gem_instdir}/extra/dynflow-debug.sh %{buildroot}%{foreman_dir}/script/foreman-debug.d/60-dynflow_debug
 
 # Logrotate script
-install -Dp -m0644 %{SOURCE1} %{buildroot}%{_root_sysconfdir}/logrotate.d/%{gem_name}
+install -Dp -m0644 %{SOURCE1} %{buildroot}%{_sysconfdir}/logrotate.d/%{gem_name}
```

Comment:
This would introduce a regression

Reply:
I wonder if we should give up on the attempt of using the `gem2rpm` template, and keep here just the changes needed for adding the npm deps: I don't feel very comfortable with the changes outside of the npm ones

Reply:
Most of those are safe and a one-time-thing. It'll make future updates easier.